### PR TITLE
[ci-app] Tweak quality gates documentation to match updated UI

### PR DESCRIPTION
**Pull request by soyguijarro**
### What does this PR do?

This PR updates CI Visibility's quality gates documentation to match the product UI which was recently updated

### Motivation

UI update PR: https://github.com/DataDog/web-ui/pull/103868

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.
- [ ] Check that `cache_enabled` is set to `true` in the `pull_config_preview.yaml` file

### CODE REVIEW

File: content/en/continuous_integration/quality_gates/_index.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ There are two main steps required to set up and use Quality Gates:
 To create Quality Gates rules for your organization, your user account must have the `quality_gate_rules_write` [permission][1].
 
 1. In Datadog, navigate to [**CI** > **Quality Gate Rules**][2] and click **+ New Rule**.
-2. Select the rule type. You can choose between `Static Analysis` and `Test`.
-3. Define the rule scope. The rule scope defines when the rule should be evaluated. For example, you can create a rule that it is evaluated only on specific repositories and branches. To define the scope for a rule, switch to `select when to evaluate` and add included or excluded values for the scope. More information on [rule scopes](#rule-scope).
+2. Define the rule scope. The rule scope defines when the rule should be evaluated. For example, you can create a rule that it is evaluated only on specific repositories and branches. To define the scope for a rule, switch to `select when to evaluate` and add included or excluded values for the scope. More information on [rule scopes](#rule-scope).
```
Comment (brett0000FF) [inline content/en/continuous_integration/quality_gates/_index.md:37]:
```suggestion
2. Define the rule scope. The rule scope defines when the rule should be evaluated. For example, you can create a rule that it is evaluated only on specific repositories and branches. To define the scope for a rule, switch to `select when to evaluate` and add included or excluded values for the scope. See [rule scopes](#rule-scope) for more information.
```

**Review by brett0000FF — CHANGES_REQUESTED** [review]
Looks good! Just one minor suggestion for the content.

Regarding the image, please note that there is a known issue where we need to update the name of the image for it to appear correctly on the site. You need to change `qg_rule_condition_sa_errors_security.png` to something like `qg_rule_condition_sa_errors_security2.png` and then update the image reference on line 43 to the new name.

Let me know if you need any assistance with that. Thanks!

**Comment by soyguijarro** [comment]
> Regarding the image, please note that there is a known issue where we need to update the name of the image for it to appear correctly on the site. You need to change `qg_rule_condition_sa_errors_security.png` to something like `qg_rule_condition_sa_errors_security2.png` and then update the image reference on line 43 to the new name.

I had no idea, thank you! Updated here: https://github.com/DataDog/documentation/pull/19666/commits/a80beeaec6c18398d08328a88be9e9f07c60f067

**Review by brett0000FF — APPROVED** [review]
LGTM. Thanks! 👍

**Pull request merged**